libs/components/src/lib/accordion-item/accordion-item.scss

```
@@ -16,10 +16,15 @@
 @use '../../../../shared/src/lib/sass/mixins/focus' as focus;
 @use '../../../../shared/src/lib/sass/mixins/focus' as focus-variables;
 
+$vvd-accordion-item-accent-firm-wrapper: --_vvd-accordion-item-accent-firm-wrapper;
+
 :host {
 	display: flex;
 	box-sizing: border-box;
 	flex-direction: column;
+	#{$vvd-accordion-item-accent-firm-wrapper}: var(
+		--vvd-accordion-item-accent-firm
+	);
 }
 
```

Maybe the ugliness can be better contained with something like this:

```suggestion
:host {
	display: flex;
	box-sizing: border-box;
	flex-direction: column;
}

@include variables-backwards-compatibility();
```

with 

```
@mixin variables-backwards-compatibility() {
  :host {
    // build vars for the connotations configured in connotation/config
  }

  .control {
    // do the mapping
  }
}
```

Will make it easier to delete once the time comes